client/src/components/Cards/ProfileCard.tsx

```diff
@@ -29,46 +30,40 @@ const useStyles = makeStyles({
 });
 
 interface ProfileCardProps {
-  photo: string;
-  name: string;
-  caption?: string;
-  rating?: number;
-  description: string;
-  location: string;
-  pay: number;
+  profile: Profile;
 }
 
-const ProfileCard: React.FC<ProfileCardProps> = ({ photo, name, caption, rating, description, location, pay }) => {
+const ProfileCard: React.FC<ProfileCardProps> = ({ profile }) => {
   const classes = useStyles();
   return (
     <Card className={classes.root}>
       <CardActionArea>
         <CardContent sx={{ display: 'flex', flexDirection: 'column', justifyItems: 'center' }}>
-          <Avatar alt="Profile Photo" src={photo} className={classes.avatar} />
+          <Avatar alt="Profile Photo" src={profile?.photo} className={classes.avatar} />
```

Comment:
As opposed to using ternary operators before accessing object properties, why don't we consider rendering the `ProfileCard ` component only if a profile exists 😄

Reply:
I removed the ternary operators. I believe ProfileCard won't render unless a Profile is passed in through props